python/pyarrow/_compute.pyx

```diff
@@ -1035,8 +1022,7 @@ class StrptimeOptions(_StrptimeOptions):
 cdef class _StrftimeOptions(FunctionOptions):
     def _set_options(self, format, locale):
         self.wrapped.reset(
-            new CStrftimeOptions(tobytes(format), tobytes(locale))
-        )
+            new CStrftimeOptions(tobytes(format), tobytes(locale)))
```

Comment:
(another styling nit: note that some formatters (eg black) will actually prefer how it was before)